cmp/grid/Grid.js

```diff
@@ -781,12 +781,14 @@ class GridLocalModel extends HoistModel {
 
         if (!record || !xhColumn) return value;
 
-        return XH.gridExportService.getExportableValueForCell({
+        const exportValue = XH.gridExportService.getExportableValueForCell({
             gridModel: model,
             record,
             column: xhColumn,
             node
         });
+
+        return exportValue?.value ?? exportValue;
```

Comment:
Good catch!

Reply:
Looks like CSV export also affected by this.  Making another tweak.

https://github.com/xh/hoist-react/issues/2797